source/Contracts/ChargeMessage/DocumentType.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ReSharper disable once CheckNamespace - Type is shared so namespace is not determined by project structure/namespace
+namespace Energinet.DataHub.Charges.Contracts.Charge
+{
+    public enum DocumentType
+    {
+        D05 = 1, // CimRequestChangeBillingMasterData;
+        D10 = 2, // CimRequestChangeOfPriceList;
```

Comment:
Let's remove the cim part

```suggestion
        D05 = 1, // RequestChangeBillingMasterData;
        D10 = 2, // RequestChangeOfPriceList;
```